pootle/apps/pootle_store/unit/filters.py

```diff
@@ -42,11 +43,14 @@ def filter_checks(self):
             return self.qs.filter(
                 qualitycheck__false_positive=False,
                 qualitycheck__name__in=self.checks).distinct()
-        elif self.category:
+        elif self.category or self.category is False:
```

Comment:
As I understand `self.category is False` was meant to be used to use no category at all (i.e. fall back to all categories), but as it turns out a) `self.category` is never assigned the `False` value and b) the default `return` which is below already takes care of the fallback.